packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -521,6 +521,67 @@ const DatePickerInput = (props) => {
     }
   };
 
+
+
+  const handleInputKeydown = (event, inputType) => {
+    const { day, month, year } = date;
+    let inputDate;
+    let formattedDate;
+    if (day.length === 2 && month.length === 2 && year.length === 4) {
+      inputDate = DateUtil.convertToISO8601(`${year}-${month}-${day}`, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
+    }
+    const validDate = DateUtil.isValidDate(formattedDate, momentDateFormat);
+
+
+    if (event.key === 't' || event.key === 'T') {
+      inputDate = DateUtil.getCurrentDate();
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
+      if (onChange) {
+        onChange(event, formattedDate);
+      }
+      const nextDayValues = DateUtil.getDateInputValues(DateUtil.dateOrder.YMD, inputDate, '-');
+      dateDispatch({ day: nextDayValues.day, month: nextDayValues.month, year: nextDayValues.year });
+      return;
+    }
+    if (event.key === '-' || event.key === '_') {
+      if (validDate) {
+        inputDate = DateUtil.decrementDateByDay(inputDate, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      } else {
+        inputDate = DateUtil.decrementDateByDay(DateUtil.getCurrentDate(), DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      }
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
+      if (onChange) {
+        onChange(event, formattedDate);
+      }
+      const nextDayValues = DateUtil.getDateInputValues(DateUtil.dateOrder.YMD, inputDate, '-');
+      dateDispatch({ day: nextDayValues.day, month: nextDayValues.month, year: nextDayValues.year });
+      return;
+    }
+    if (event.key === '=' || event.key === '+') {
+      if (validDate) {
+        inputDate = DateUtil.incrementDateByDay(inputDate, DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      } else {
+        inputDate = DateUtil.incrementDateByDay(DateUtil.getCurrentDate(), DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      }
+      formattedDate = DateUtil.strictFormatISODate(inputDate, momentDateFormat);
```

Comment:
Can you add a comment before each `if` statement about what each block does? Something like 
```js
// jump to current date
if (event.key === 't' || event.key === 'T') {
```

Reply:
Added here https://github.com/cerner/terra-framework/pull/1507/commits/69edd85edb2fc48fdfa70c5799fe8c7284906461